gigascience/gigadb-website
# feature:456 update datacite schema

# Pull request for issue: #456 

This is a pull request for the following functionalities:

Improve DataCite metadata by migrating to Datacite version 4.6 + order authors by rank (for dataset as xml) 

## How to test?

connect as admin
go to Dataset page
click "mint DOI" 
you should see the xml saved following Datacite schema version 4.6

## How have functionalities been implemented?

## Any issues with implementation?

## Any changes to automated tests?

test added

## Any changes to documentation?

## Any technical debt repayment?

## Any improvements to CI/CD pipeline?
